blog_post.md

```diff
@@ -140,18 +140,14 @@ where the last equality is a general result for geometric series. This means tha
     - reward clipping -->
 
 ## Experimental setup
+We try to follow the experimental setup from the [DQN paper](https://www.cs.toronto.edu/~vmnih/docs/dqn.pdf) closely. The loss term we use is the mean-squared error which is clipped in the range [-1; 1], as it was reported to improve the stability of the algorithm. The error is optimized by [Adam](https://arxiv.org/pdf/1412.6980.pdf) with a learning rate $$\alpha = 0.001$$. The choice of optimizer deviates from the original paper but has shown great success in deep learning recently. Additional experiments with different values of the learning rate and the contribution of error clipping are left for future work.
 
-We are now able to quantify divergence, which gives us a straightforward way to compare different algorithms in terms of divergence. We run each setup X times, and report the fraction of runs at which soft divergence occurs. We set the amount of runs to ?? to ensure statistically significant results, while taking our computational budget into account.
+Since divergence can now be quantified, we use it as a metric to compare the improvement that each of the tricks brings to DQN. We run each experiment 25 times with 25 different random seeds to achieve statistically significant results. We chose this number to be as high as possible taking into account our computational budget as well. We judge whether the model has diverged by looking at the maximal Q-value predicted in the last step of the last episode - if it is above the threshold $$\frac{1}{1-\gamma}$$, we say it has diverged. At the end, we compare the configurations by counting how many times each of them has diverged.
```

Comment:
Need to add:
- we compare qualitatively by looking at the distributions of max abs q values.
- and also quantitatively with fraction.